cube/src/main/java/org/jboss/arquillian/ce/cube/CEEnvironmentProcessor.java

```diff
@@ -61,25 +60,26 @@
 /**
  * CEEnvironmentProcessor
  * <p>
- * Temporary class to handle @Template and @OpenShiftResource annotations on
- * test classes. Eventually, these will be migrated to Cube types, at which
- * point this will delegate to those for setup/teardown (via
+ * Temporary class to handle @Template, @TemplateResources, and @OpenShiftResource 
+ * annotations on test classes. Eventually, these will be migrated to Cube types, 
+ * at which point this will delegate to those for setup/teardown (via
  * StartCube/StopCube).
  *
  * @author Rob Cernich
+ * @author Kyle Liberti
  */
 public class CEEnvironmentProcessor {
 
     private final Logger log = Logger.getLogger(CEEnvironmentProcessor.class.getName());
-
-    public interface TemplateDetails {
-        List<? extends OpenShiftResource> getResources();
+    public static interface TemplateDetails {
+    	 ArrayList<List<? extends OpenShiftResource>> getResources();
```

Comment:
Why not return a generic List instead of returning a specific implementation (ArrayList)?